src/main/java/net/dv8tion/jda/internal/requests/restaction/CommandCreateActionImpl.java

```diff
@@ -116,30 +134,64 @@ public CommandCreateAction timeout(long timeout, @Nonnull TimeUnit unit)
     @Override
     public CommandCreateAction setName(@Nonnull String name)
     {
-        Checks.notEmpty(name, "Name");
-        Checks.notLonger(name, 32, "Name");
-        Checks.matches(name, Checks.ALPHANUMERIC_WITH_DASH, "Name");
         data.setName(name);
         return this;
     }
 
+    @Nonnull
+    @Override
+    public CommandCreateAction setNameLocalization(@Nonnull DiscordLocale locale, @Nonnull String name)
+    {
+        data.setNameLocalization(locale, name);
+        return this;
+    }
+
+    @Nonnull
+    @Override
+    public CommandCreateAction setNameLocalizations(@Nonnull Map<DiscordLocale, String> map)
+    {
+        data.setNameLocalizations(map);
+        return this;
+    }
+
     @Nonnull
     @Override
     public CommandCreateAction setDescription(@Nonnull String description)
     {
-        Checks.notEmpty(description, "Description");
-        Checks.notLonger(description, 100, "Description");
         data.setDescription(description);
```

Comment:
Why are these checks gone now?

Reply:
They are already done in data#setDescription